### PR TITLE
fix: APP-474 tebu factors UI

### PR DESCRIPTION
**Pull request by blushi**
## Description

https://regennetwork.atlassian.net/browse/APP-474

Also fixing some other UI issues I’ve noticed:
- All border colors were too light
- wrong border radius around the tebu or compliance info

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by erikalogie** [comment]
> @S4mmyb you can have a look at those projects for a lower restoration percentage: https://deploy-preview-2548--terrasos.netlify.app/project/354637ec-a687-11ef-b1b2-022e215a294f https://deploy-preview-2548--terrasos.netlify.app/project/faccad54-a694-11ef-8497-022e215a294f I've fixed the UI issues you mentioned with the pie labels/lines.

Everything LGTM

File: web-marketplace/src/components/templates/ProjectDetails/TerrasosCreditsInfo/TerrasosCreditsInfo.tsx [hunk 1]
```diff
@@ -44,10 +46,12 @@ export default function TerrasosCreditsInfo({
       <TableTabs
         sx={{
           px: { xs: 0, sm: 0 },
-          pt: { xs: '50px', sm: '50px' },
+          pt: { xs: pxToRem(50), sm: pxToRem(50) },
           pb: { xs: 0, sm: 0 },
         }}
         tabs={tabs}
+        // eslint-disable-next-line lingui/no-unlocalized-strings
+        tabOuterClassName="border-sc-card-standard-stroke rounded-b-t-[10px]"
```
Comment (flagrede) [inline web-marketplace/src/components/templates/ProjectDetails/TerrasosCreditsInfo/TerrasosCreditsInfo.tsx:54]:
```suggestion
        tabOuterClassName="border-sc-card-standard-stroke rounded-b-t-10"
```

**Comment by S4mmyb** [comment]
> > @S4mmyb you can have a look at those projects for a lower restoration percentage: https://deploy-preview-2548--terrasos.netlify.app/project/354637ec-a687-11ef-b1b2-022e215a294f https://deploy-preview-2548--terrasos.netlify.app/project/faccad54-a694-11ef-8497-022e215a294f I've fixed the UI issues you mentioned with the pie labels/lines.
> 
> Everything LGTM

Agreed, thanks for testing different values!

File: web-marketplace/src/components/templates/ProjectDetails/TerrasosCreditsInfo/TerrasosCreditsInfo.TebuInfo.tsx [hunk 2]
```diff
@@ -62,51 +61,26 @@ const TebuInfo: React.FC<TebuInfoProps> = ({
   const socialCulturalIndex = projectMetadata?.['regen:socialCulturalIndex'];
   const socialCulturalCard =
     false && getSocialCulturalCard(_, SocialCulturalValueType.High);
```
[the remark this answers is not in the document]
Comment (blushi) [reply web-marketplace/src/components/templates/ProjectDetails/TerrasosCreditsInfo/TerrasosCreditsInfo.TebuInfo.tsx:63]: I'm not sure, I do see social and cultural cards in the figma components: https://www.figma.com/design/brkxGV5qNOkZUp0YQl1cxO/Terrasos-Phase-1?node-id=1295-242275&node-type=frame&m=dev
but they are not on the project page view on figma
is there a reason or is this an oversight? @erikalogie @S4mmyb

[the remark this answers is not in the document]
Comment (blushi) [reply web-marketplace/src/components/templates/ProjectDetails/TerrasosCreditsInfo/TerrasosCreditsInfo.TebuInfo.tsx:63]: looking at the terrasos data spreadsheet I don't see any relevant data for Social Cultural Index so maybe it's fine for now

[the remark this answers is not in the document]
Comment (blushi) [reply web-marketplace/src/components/templates/ProjectDetails/TerrasosCreditsInfo/TerrasosCreditsInfo.TebuInfo.tsx:63]: ok looks like @wgardiner didn't implement the different tooltip and disabled showing the social index, should we have a separate task that we handle once we have this type of projects coming in or should I address it here?

[the remark this answers is not in the document]
Comment (erikalogie) [reply web-marketplace/src/components/templates/ProjectDetails/TerrasosCreditsInfo/TerrasosCreditsInfo.TebuInfo.tsx:63]: I think we should open a separate task to make it work for 4.0 projects

[the remark this answers is not in the document]
Comment (blushi) [reply web-marketplace/src/components/templates/ProjectDetails/TerrasosCreditsInfo/TerrasosCreditsInfo.TebuInfo.tsx:63]: ok I'll do that and merge the current PR

**Pull request merged**